roles/edpm_bootstrap/defaults/main.yml

```diff
@@ -80,3 +82,21 @@ edpm_bootstrap_command: ""
 # The path of the file that will be used as a marker when the edpm_bootstrap_command
 # has been executed on the node.
 edpm_bootstrap_command_done_file: /.bootstrap_command_done
+
+# edpm_fips_mode is used to check fips status or enable/disable it
```

Comment:
edpm_bootstrap_fips_mode?

Reply:
Id like to use `edpm_fips_mode` as a "top level" var in order to be shared between roles but i agree that I can improve the description of the local var here.